test/integration/fakebroker.go

```diff
@@ -0,0 +1,165 @@
+/*
```

Comment:
I think this file should be called fakebrokerclient.go or something like fakeOSBClientWrapper FakeOSBClientHooker